frontend/src/components/Board/Board.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
```

Comment:
Zmieniłbym nazwę tego pliku. Board.jsx jest zbyt ogólne jak dla mnie. Kojarzy mi się bardziej z widokiem, niż z komponentem. Może BoardItem.jsx?

Reply:
Robi się

Reply:
albo jednak nie bo mi intellij jakies bledy wywala, potrzebuje to zmergowac ASAP i po tym dac commit szybki zeby to w ogole dodac do app.js i takie rzeczy naprawic, bo ja zpullowalem maina ale tyle jakis dziwnych rzeczy mi sie robi ze wolalbym pobrac od nowa projekt caly

Reply:
Chyba nie musimy importować reacta?